### ISSUE TITLE
Yolov8 load "yolov8n.pt" even if I set "yolov8m.pt" as the model

### ISSUE THREAD

**Issue by lucas-mior** [ultralytics/ultralytics#2698]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

For this setup:
```
model = YOLO("yolov8m.pt")
model.train(data="./custom_data.yaml", epochs=1, imgsz=640, batch=4, cache="disk")
```
The output clearly shows the `"yolov8n.pt"` being downloaded and used for training.

### Minimal Reproducible Example
```
yolo train model=yolov8m.pt
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@lucas-mior YOLOv8n is used for AMP checks prior to training start to decide whether to allow this training mode (as your console printout clearly displays). Your YOLOv8m model will train as normal.

Removing bug label. Please do not raise bug reports for questions.
